onepwd/plugins/modules/item.py

```diff
@@ -6,7 +6,7 @@
 description:
   - Create 1Password items
   - Add and remove fields
-version_added: 2.12.3
+version_added: 2.12.4
```

Comment:
This adds a new feature so I think it should be a minor version instead of a hotfix